server/src/main/java/org/apache/iotdb/db/metadata/mnode/UnaryMeasurementMNode.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.db.metadata.mnode;
+
+import org.apache.iotdb.tsfile.file.metadata.enums.TSDataType;
+import org.apache.iotdb.tsfile.write.schema.MeasurementSchema;
+
+/** Represents an MNode which has a Measurement or Sensor attached to it. */
+public class UnaryMeasurementMNode extends MeasurementMNode {
+
+  /** measurement's Schema for one timeseries represented by current leaf node */
+  private MeasurementSchema schema;
```

Comment:
```suggestion
  private UnaryMeasurementSchema schema;
```

Reply:
Fixed. There will be a large quantity of codes for replacing.